navigation.yaml

```diff
@@ -109,7 +109,7 @@ desktop:
     - title: For partners
       path: /desktop/partners
     - title: Ubuntu in organisations
-      path: /desktop/ubuntu-in-organisations
+      path: /desktop/organisations
```

Comment:
@frankiegranato I don't think this scans well

![image](https://user-images.githubusercontent.com/441217/39036883-d00f1f8e-4476-11e8-95ba-f5e438985241.png)

could it be 'In organisations' or 'For organisations' ?

Reply:
I went with 'For...' to match the others